### DISCUSSION TITLE
New Mojo version this week?

### DISCUSSION THREAD

**Discussion by gryznar** [modular/modular#549] [Q&A] [answered]
Is new version going to be published this week?

**Comment by PriNova** [comment]
Since yesterday the version 0.2.1 is released.

**Reply by gryznar** [reply]
So i hope that there will be also official changelog here: https://docs.modular.com/mojo/changelog.html

**Reply by PriNova** [reply]
Maybe it is only bug fix patch and nothing semantic new.

**Reply by gryznar** [reply]
Maybe. But still it is kinda weird, because previously was notification if there is / is not new version every week.

**Reply by PriNova** [reply, marked as answer]
At least in the SDK version. I updated yesterday and already filed an issue because of complete broken Mojo #558

They mention also only publishing updates to the change log if it has not only internal implementations

**Reply by gryznar** [reply]
Thank you for clarification :)

**Reply by scottamain** [reply]
The SDK is in beta for a limited number of participants, so we are not publishing changelogs for the beta. When we release the SDK publicly, we will update the changelog with everything that has changed since the last public Mojo release.
